README.md

```diff
@@ -48,19 +48,42 @@ Giving the following output
 
 ## Example: Counting incoming Spanish traffic
 
+Create file /etc/geoip.nft
 ```
-  table filter {
-    include "./geoip-def-all.nft"
-    include "./geoip-ipv4.nft"
-    include "./geoip-ipv6.nft"
+  #!/usr/sbin/nft -f
+
+  table geoip {
```

Comment:
Creating a separate table to contain the chain for geoip mapping is interesting.